html/changelog.html

```diff
@@ -1717,6 +1717,99 @@ <h3 class="author">PrimeDSS13 updated:</h3>
 			<ul class="changes bgimages16">
 				<li class="bugfix">Packed: tightens out some knots so power flowers again</li>
 			</ul>
+			<h2 class="date">2020.11.30</h2>
```

Comment:
probably not necessary to include this

Reply:
You're actually not supposed to directly edit this file *at all*.

Reply:
Oh wait you didn't
Why is it here then?

Reply:
gitfu gone wrong

Reply:
still gotta fix this